### PR TITLE
SAIC-75 Now can migrate user without email

### PR DESCRIPTION
**Pull request by Settis**

### CODE REVIEW

**Comment by Settis** [comment]
Add fix for crating user without TenantID.

**Comment by roman-verchikov** [comment]
Looks good, +1

**Comment by miarmak** [comment]
Looks good. +1

File: cloudferrylib/os/identity/keystone.py [hunk 1]
```diff
@@ -165,12 +166,12 @@ def read_info(self, **kwargs):
         has_roles_by_ids_cached = self._get_user_roles_cached()
         for user in user_list:
             usr = self.convert(user, self.config)
-            if has_tenants_by_id_cached(user.tenantId):
+            if has_tenants_by_id_cached(getattr(user, 'tenantId', '')):
```
Comment (ntymtsiv) [inline cloudferrylib/os/identity/keystone.py]: With this condition users without tenant id won't be migrated.

**Pull request merged**